analytics-api/src/analytics_api/resources/survey_result.py

```diff
@@ -30,17 +32,24 @@
 
 
 @cors_preflight('GET,OPTIONS')
-@API.route('/<engagement_id>')
+@API.route('/<engagement_id>/<dashboard_type>')
```

Comment:
maybe it's clearer if we just create two endpoints one /public and one /internal instead of having a path variabl